src/openfermion/resource_estimates/__init__.py

```diff
@@ -10,10 +9,10 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
-#coverage: ignore
-import pytest
 
 try:
+    import jax
     import pyscf
-except ImportError:
-    pytest.skip('Need pyscf for resource estimates', allow_module_level=True)
+    HAVE_DEPS_FOR_RESOURCE_ESTIMATES = True
+except ModuleNotFoundError:
+    HAVE_DEPS_FOR_RESOURCE_ESTIMATES = False
```

Comment:
should we re-raise the exception with a helpful error message? presumably if these imports fail something else will give a weird error message when you actually go and try to use any of the functionality

Reply:
I don't think so, wouldn't this cause an import error (pytest will error rather than skip the tests)